### ISSUE TITLE
a11y: Color Contrast - Projects/Namespaces

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13740]
Insufficient color contrast for the borders of form controls. Insufficient color contrast for standard text

## Actual result

Some form control borders have insufficient contrast with their background, making it difficult for people with low vision and color blindness to distinguish them.

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio: 1.34:1
Main Content
- Filter input field

Charts
Main Content
- Filter input field"

"Foreground color: #FFFFFF
Background color: #3D98D3
Contrast ratio: 3.16:1
Main content
- Create Namespace, Create Project, Move, Download YAML, Delete (on default state)
- Move, Edit Config, Edit YAML and so on (on hover - which gets available on activating ""More actions"" button)

Foreground color: #FFFFFF
Background color: #297DB4
Contrast ratio: 4.48:1 
Main content
- Create Namespace, Create Project (on focus state)

Foreground color: #BEC1D2
Background color: #FFFFFF
Contrast ratio: 1.78:1
Main content
- Filter placeholder text

Foreground color: #3D98D3
Background color: #F2F2F2
Contrast ratio: 2.82:1
Main content
- Create Namespace in table (on default)  

Foreground color: #5D995D
Background color: #FFFFFF
Contrast ratio: 3.39:1
Main content
- Active (in not selected state) 

Foreground color: #5D995D
Background color: #E9EEF1
Contrast ratio: 2.90:1
Main content
- Active (in selected state) 

Foreground color: #3D98D3
Background color: #E9EEF1
Contrast ratio: 2.70:1
Main content
- default, local, user-2rgrp (on selected checkbox) 

Foreground color: #3D98D3
Background color: #F4F5FA
Contrast ratio: 2.90:1
Main content
- default, local, user-2rgrp (on hover) 

"

## Expected result

The contrast ratio between the foreground border color of the form field and background color should be sufficient.

The contrast ratio between foreground text and background for the standard text should be sufficient.

**Comment by rak-phillip** [comment]
Issue 149 fails VPAT. No notes. Original comment:

> Some form control borders have insufficient contrast with their background, making it difficult for people with low vision and color blindness to distinguish them.

Issue 150 fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
>
> Instances:
> Foreground color: #BEC1D2
> Background color: #FFFFFF
> Contrast ratio: 1.78:1 
> Main Content: 
> - Filter
> 
> Foreground Text Color: #2F68DF
> Background Text Color: #EDF2FC
> Ratio: 4.4:1
> Main Content
> - Create Namespace (on hover and focus)
> 
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text."

Issue 151 fails VPAT:

> "Actual Result:
> The informative image present on the page has an insufficient color contrast ratio with their background
> making it difficult for low vision users to discern meaningful graphics that have insufficient contrast.
> 
> Instance:
> Foreground color: #B0B1B6
> Background color: #F4F5FA
> Contrast ratio: 1.96:1 
> Main Content 
> - Sort image buttons (in not selected state)
>  
> Recommendation
> For the key parts of all the informative images, ensure that the contrast between foreground, background or adjacent colors is 3:1 to make the image's information accessible for low vision and color-blind users.
> 
>  "
